app/assets/stylesheets/administrate/application.scss

```diff
@@ -10,3 +10,5 @@
 @import "library/**/*";
 @import "base/**/*";
 @import "components/**/*";
+
+@import "administrate/customize";
```

Comment:
Interesting. I think this is surfacing a larger issue, which is that Administrate comes packaged with `sass-rails`, but the example app is using Sprockets in its main stylesheets manifest. Sprockets and `sass-rails` handle imports differently.

So, if we are letting Sass (`sass-rails`) handle the importing, the import statement I posted above should work. I pulled down this PR, and when I run the app locally with the current import statement, nothing I put in `_customizations.scss` is compiled into the main CSS file. Changing it to `@import "customizations";` does work.